tensorboard/components/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -16,56 +16,52 @@
 -->
 
 <link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="tf-scalar-chart.html">
 <link rel="import" href="tf-smoothing-input.html">
+<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel="import" href="../paper-item/paper-item.html">
+<link rel="import" href="../paper-menu/paper-menu.html">
 <link rel="import" href="../tf-backend/tf-backend.html">
-<link rel="import" href="../tf-color-scale/tf-color-scale.html">
 <link rel="import" href="../tf-dashboard-common/tf-dashboard.html">
 <link rel="import" href="../tf-dashboard-common/tf-option-selector.html">
-<link rel="import" href="../tf-dashboard-common/tf-panes-helper.html">
-<link rel="import" href="../tf-runs-selector/tf-runs-selector.html">
 <link rel="import" href="../tf-imports/lodash.html">
-<link rel="import" href="../vz-line-chart/vz-line-chart.html">
-<link rel="import" href="../iron-collapse/iron-collapse.html">
-<link rel="import" href="../paper-dropdown-menu/paper-dropdown-menu.html">
-<link rel="import" href="../paper-icon-button/paper-icon-button.html">
-<link rel="import" href="../paper-menu/paper-menu.html">
-<link rel="import" href="../paper-item/paper-item.html">
+<link rel="import" href="../tf-runs-selector/tf-runs-selector.html">
 
 <!--
-tf-scalar-dashboard is a complete frontend that loads runs from a backend,
-and creates chart panes that display data for those runs.
-
-It provides a categorizer, run selector, and x type selector, by which the user
-can customize how data is organized and displayed.
-
-Each chart has a button that can toggle whether it is "expanded"; expanded
-charts are larger.
+  A frontend that displays a set of tf-scalar-charts, each of which
+  represents the time series for a particular tag. This dashboard
+  provides a categorizer, run selector, and abcissa selector, by which
+  the user can customize how data is organized and displayed.
 -->
 <dom-module id="tf-scalar-dashboard">
   <template>
     <tf-dashboard-layout>
       <div class="sidebar">
         <div class="sidebar-section">
+          <!--
+            TODO(wchargin): What do we want to do about the categorizer?
```

Comment:
I like the idea of replacing it. I think it is too complex, and patterns around building persistent state in the UI are a poor match for a service without a persistent backend.